### PR TITLE
Move CatonFixture in its own bazel module

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**

### CODE REVIEW

File: daml-script/test/src/test-utils/com/daml/lf/engine/script/test/CompiledDar.scala
Comment (samuel-williams-da) [inline daml-script/test/src/test-utils/com/daml/lf/engine/script/test/CompiledDar.scala:1]: We now have this file twice in 2 places, should it maybe go somewhere else?

**Review by samuel-williams-da — APPROVED** [review]

**Comment by remyhaemmerle-da** [comment]
@samuel-williams-da @carlpulley-da This will conflict with you migration PRs. 
Please take care to either:
1. merge your PRs before this one
2. rebase you PRs with main if you merge afrer this one. 

In case of 1, please ping me once you merged.

**Comment by samuel-williams-da** [comment]
I'm happy to rebase though if you'd like to merge now, given i believe stephen is in NY

**Comment by carlpulley-da** [comment]
Merged 16811 and 16829.

Waiting on CI with 16836.

16837 is in the queue after 16836.

All that said, changes here look straightforward, so I'm happy to rebase once you merge this PR in.

**Pull request merged**
